workflows/process_slc_pair/ConvertToTif.py

```diff
@@ -15,18 +15,24 @@ class ConvertToTif(luigi.Task):
   sourceSRS = luigi.Parameter()
   outputSRS = luigi.Parameter()
 
-  def convert(self, input, outputFolder, outputFilePattern, sourceSRS, outputSRS, runAsShell=True):
-    outputFullPath = os.path.join(outputFolder, '{0}.tif'.format(outputFilePattern))
+  def convertToCog(self, input, outputFilePath, runAsShell=True):
+    log.info('Creating output GeoTIFF at {0}'.format(outputFilePath))
 
-    log.info('Creating output GeoTIFF at {0}'.format(outputFullPath))
+    cmd = f'gdal_translate {input} {outputFilePath} -of COG -co COMPRESS=LZW'
+    log.info('Running {0}'.format(cmd))
 
-    cmd = 'gdalwarp -s_srs EPSG:{0} -t_srs EPSG:{1} -dstnodata 0 -r near -of GTiff -tr 10.0 10.0 -co "COMPRESS=DEFLATE" {2} {3}'.format(sourceSRS, outputSRS, input, outputFullPath)
+    return subprocess.run(cmd, shell=runAsShell)
+  
+  def reproject(self, input, outputFilePath, sourceSRS, outputSRS, runAsShell=True):
+    log.info('Creating output GeoTIFF at {0}'.format(outputFilePath))
+
+    cmd = 'gdalwarp -s_srs EPSG:{0} -t_srs EPSG:{1} -dstnodata 0 -r near -of GTiff -tr 10.0 10.0 -co "COMPRESS=DEFLATE" {2} {3}'.format(sourceSRS, outputSRS, input, outputFilePath)
     log.info('Running {0}'.format(cmd))
 
     return subprocess.run(cmd, shell=runAsShell)
 
-  def getInputFile(self, outputFolderWithPattern):
-    dataFolder = '{0}.data'.format(outputFolderWithPattern)
+  def getInputFile(self, workingFolderWithPattern):
+    dataFolder = '{0}.data'.format(workingFolderWithPattern)
```

Comment:
Changes to get it to write to the working folder instead of directly to the output folder.